app/lib/fhir_mapper/patient.rb

```diff
@@ -22,7 +22,7 @@ def fhir_record(reference_id: nil)
         name: [FHIR::HumanName.new(family: family_name, given: given_name)],
         birthDate: date_of_birth&.strftime("%Y-%m-%d"),
         gender: gender_fhir_value,
-        address: [FHIR::Address.new(postalCode: address_postcode)]
+        address: [FHIR::Address.new(postalCode: address_postcode || "ZZ99 3CZ")]
```

Comment:
I think it would be worth extracting the `"ZZ99 3CZ"` literal to a constant with a name explaining what it is.